src/setup/testnet/token1.txt

```diff
@@ -0,0 +1,8 @@
+eyJtYW5pZmVzdCI6IkpBQUFBQUZ4SWUyUDhJZjJTTlFvL3MzZDZReDl6Wld3cVZyWkc1N3Vh
```

Comment:
IMO these tokens are small enough that we could keep them in the source files together with the keys, but I don't mind that much

Reply:
Let's leave it as is and we'll come back to this maybe later.